src/detectors/typeChecker/FileLinter.ts

```diff
@@ -211,6 +212,69 @@ export default class FileLinter {
 		});
 	}
 
+	getSymbolModuleDeclaration(symbol: ts.Symbol) {
+		let parent = symbol.valueDeclaration?.parent;
+		while (parent && !ts.isModuleDeclaration(parent)) {
+			parent = parent.parent;
+		}
+		return parent;
+	}
+
+	analyzeLibInitCall(node: ts.CallExpression) {
+		const nodeExp = (
```

Comment:
(style) we usually just check whether a node is of one of the expected types and stop (return statement) otherwise. This might be easier to read and consistent with the other code in this file.